docs/src/modules/components/AppFrame.js

```diff
@@ -114,7 +114,7 @@ const styles = (theme) => ({
   },
   appBar: {
     color: theme.palette.mode === 'light' ? null : '#fff',
-    backgroundColor: theme.palette.mode === 'light' ? null : theme.palette.background.level2,
+    backgroundColor: theme.palette.mode === 'light' ? null : theme.palette.background.level1,
```

Comment:
reduce the background color of AppBar. no strong opinion.

**Before**
![image](https://user-images.githubusercontent.com/18292247/120280451-e2986380-c2e1-11eb-93e7-990efc8df665.png)

**New**
![image](https://user-images.githubusercontent.com/18292247/120280506-f2b04300-c2e1-11eb-8ac6-2255bd664df5.png)

Reply:
As a side note, It would be great if we could get ride of level1 and level2, using the built-in API, or add a new one if missing.